### PR TITLE
Improved first load time and UX on first load

### PR DESCRIPTION
**Pull request by paralainer**
Two changes:
* doing requests in parallel on first load
* showing bouncer before js bundle is loaded

First load on slow internet is pretty slow, and also blank screen while it loading js bundle doesn't help, so decided to improve that.

Also have an idea to load comments before we actually loaded iframe and pass a message inside iframe on load, but this is more complicated change, its better to split them, so probably next PR (if tests will show that it makes sense)

### CODE REVIEW

File: web/app/components/root/root.jsx [hunk 1]
```diff
@@ -51,28 +51,25 @@ export default class Root extends Component {
...
-          });
-
-        api.getConfig().then(config => {
+    api.getConfig().then(config => {
           store.set('config', config);
           this.setState({ config });
-        })
-      });
+    });
+    Promise.all([
```
Comment (paralainer) [inline web/app/components/root/root.jsx]: It seems that we can do that these requests in parallel, there is no need to wait result of first to make a second

File: web/iframe.html [hunk 2]
```diff
@@ -13,10 +13,42 @@
...
+        0% {transform: scale(0);}
+        40% {transform: scale(1);}
+        80%, 100% {transform: scale(0);}
+    }
   </style>
 </head>
 <body>
-  <div id="remark42"></div>
+  <div id="remark42">
+      <div class="iframe_preloader">
```
Comment (paralainer) [inline web/iframe.html]: Basically I am duplicating code here and in css. As code is pretty simple IMO its not a big deal, but if you have an idea of how to deduplicate it I am happy to implement.

Comment (igoradamenko) [reply web/iframe.html]: It's fine. I will try to optimise in the nearest future.

**Comment by igoradamenko** [comment]
LGTM.

**Pull request merged**